refinery/selenium_testing/tests.py

```diff
@@ -1,23 +1,26 @@
+import os
+
 from django.contrib.auth.models import User
 from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+
 from pyvirtualdisplay import Display
 from selenium import webdriver
+from unittest import skipIf
 
 from core.management.commands.create_public_group import create_public_group
 from core.management.commands.create_user import init_user
 from core.models import Analysis, DataSet
 
 from factory_boy.utils import make_analyses_with_single_dataset, make_datasets
 from selenium_testing.utils import (
-    assert_body_text, login, wait_until_id_clickable, MAX_WAIT,
-    assert_text_within_id, delete_from_ui, wait_until_class_visible,
-    wait_until_id_visible)
-
-# Start a pyvirtualdisplay for geckodriver to interact with
-display = Display(visible=0, size=(1900, 1080))
-display.start()
+    assert_body_text, assert_text_within_id, delete_from_ui, login,
+    MAX_WAIT, wait_until_class_visible, wait_until_id_clickable,
+    wait_until_id_visible
+)
 
 
+@skipIf("prod" not in os.getenv("DJANGO_SETTINGS_MODULE"),
```

Comment:
@hackdna We don't currently have way to run unit tests separately from Selenium tests. Sure, you can run individual app's test suites, but this isn't a good solution, especially in the case of Refinery, because our apps are tightly coupled and changes in one app can break tests in another.

I think this solution is simple and works fine. If one wants to explicitly run selenium tests locally, switch to prod mode. Otherwise, selenium based tests are run on Travis every time you push.

As for Django 1.10, yes I mentioned that tagging feature to you... but again the ability to use that is out of our reach for awhile.